docker/Dockerfile

```diff
@@ -1,14 +1,16 @@
-ARG BASEIMAGE=us.gcr.io/k8s-artifacts-prod/build-image/debian-base-amd64:buster-v1.7.0
+ARG BASEIMAGE=us.gcr.io/k8s-artifacts-prod/build-image/debian-base:buster-v1.7.0
 
 FROM golang:1.16 as builder
 WORKDIR /go/src/sigs.k8s.io/secrets-store-csi-driver
 ADD . .
 ARG TARGETARCH
 ARG TARGETOS
-ARG LDFLAGS
-ARG GOPROXY
-ENV GOPROXY=${GOPROXY}
-RUN CGO_ENABLED=0 GOOS=$TARGETOS GOARCH=$TARGETARCH go build -a -ldflags "${LDFLAGS}" -o _output/secrets-store-csi ./cmd/secrets-store-csi-driver
+ARG TARGETPLATFORM
+
+RUN export GOOS=$TARGETOS && \
+    export GOARCH=$TARGETARCH && \
+    export GOARM=$(echo ${TARGETPLATFORM} | cut -d / -f3 | tr -d 'v') && \
```

Comment:
you don't need this if its just arm64 https://github.com/golang/go/wiki/GoArm

Reply:
I was testing `arm/v7` so added it. I'll remove it.

Reply:
Done!